UI/window-basic-main-outputs.cpp

```diff
@@ -2060,7 +2114,6 @@ inline void AdvancedOutput::SetupVodTrack(obs_service_t *service)
 		if (!ServiceSupportsVodTrack(service))
 			vodTrackEnabled = false;
 	}
-
 	if (vodTrackEnabled && streamTrack != vodTrackIndex)
```

Comment:
`int streamTrack` in this function is overshadowing `OBSEncoder streamTrack` used in the PR

Reply:
Thanks for catching this. `int streamTrack` is named int streamTrackIndex in other places; so I've renamed it accordingly in the two places where it's used.